### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ianschmitz**

### CODE REVIEW

File: samples/basic-component/src/components/BasicComponent/BasicComponent.tsx [hunk 1]
```diff
@@ -23,7 +23,10 @@ const BasicComponent = (
                 I am a basic component
             </Typography>
             {/* Change the underlying model property value to trigger a re-render */}
-            <Button onClick={() => (model.hidden = !model.hidden)}>
+            <Button
+                data-test="BasicComponent-btn"
```
Comment (connerleverett) [inline samples/basic-component/src/components/BasicComponent/BasicComponent.tsx:27]: @eric-parsons I added this for testing purposes. Just wanted to make sure you're OK with it.

Comment (eric-parsons) [reply samples/basic-component/src/components/BasicComponent/BasicComponent.tsx:27]: It's not a showstopper, but I'm curious why it's needed?

Comment (connerleverett) [reply samples/basic-component/src/components/BasicComponent/BasicComponent.tsx:27]: The test for this component was failing. Adding a `wait(1000)` or targeting the element with the data-test worked and I thought targeting the element would be more stable.

**Comment by connerleverett** [comment]
This is on hold till release of 5.13 as the unit tests on the master branch fail on the build server.

**Pull request merged**